cmd/ops_agent_windows/run_windows.go

```diff
@@ -109,21 +128,21 @@ func (s *service) checkForStandaloneAgents(unified *confgenerator.UnifiedConfig)
 	return nil
 }
 
-func (s *service) generateConfigs() error {
+func (s *service) generateConfigs() (confgenerator.UnifiedConfig, error) {
```

Comment:
It would probably be better if this returned a pointer to `confgenerator.UnifiedConfig`. `CollectOpsAgentSelfMetrics` already takes a pointed to it anyway, and you get the added benefit of returning `nil` instead of `confgenerator.UnifiedConfig{}`